README.md

```diff
@@ -63,22 +37,4 @@ new Vue({
 - show/hide different elements to avoid noise & keep the user focused.
 - support up to 3 levels deep on nested keys.
 - support all laravel different translation key formats.
-- directly copy translation key by clicking on the tool-tip.
-
-<br>
-
-## Usage
-
-- for styling we use ***bulma***
-
-> ***Or Use another Framework***
```

Comment:
not sure why ppl get confused with this, but u r not the first one to fall into this quote trap.